src/site/content/en/fast/signed-exchanges/index.md

```diff
@@ -0,0 +1,328 @@
+---
+layout: post
+title: Signed Exchanges (SXGs) overview and guide
```

Comment:
I added `overview` and `guide` because I want to make it clear what kind of content the reader is getting.
[Divio's Documentation System](https://documentation.divio.com/introduction/#the-secret) does a good job explaining why we want to structure our docs according to the four common types of documentation. I have been lax about this in the past but I'm going to start enforcing it more on the site.

Also, when I see a title that only says `Signed Exchanges` it's unclear what type of documentation I'm getting. Is it a reference? Guide? Overview?

Another option is to follow the content types more closely and actually split this content up into an overview and a guide. I tried doing that but it's a little awkward because there isn't much conceptual information. If you think it works, though, I'm open to this approach.

CC @mihajlija